packages/kbn-custom-events/src/fullstory.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { sha256 } from 'js-sha256'; // loaded here to reduce page load bundle size when FullStory is disabled
```

Comment:
I'd probably write this file in JS and put a .d.ts next to it to define the public types